data/moves.ts

```diff
@@ -5390,6 +5390,17 @@ export const Moves: {[moveid: string]: MoveData} = {
 			if (!this.runEvent('ChargeMove', attacker, defender, move)) {
 				return;
 			}
+
+			// In SwSh, Fly's animation leaks the initial target through a camera focus
+			// The animation leak target itself isn't "accurate"; the target it reveals is as if Fly weren't a charge movee
+			// (Fly, like all other charge moves, will actually target slots on its charging turn, relevant for things like Follow Me)
+			// We use a generic single-target move to represent this
+			if (this.gameType === 'doubles') {
+				const animatedTarget = attacker.getMoveTargets(this.dex.getActiveMove('tackle'), defender).targets[0];
```

Comment:
so if someone uses normalize fly and then somehow obtains stalwart, what happens here with ion deluge and lightning rod

Reply:
> so if someone uses normalize fly and then somehow obtains stalwart, what happens here with ion deluge and lightning rod

This one doesn't need to be worried about, because you can't have Normalize and Stalwart simultenously. This only matters on the first turn of Fly.

Reply:
Tested with Mold Breaker Electrify + Fly into LightningRod, used Fly into LightningRod's partner, and the animation showed Fly targeting the LightningRod Pokemon. Of course, on the way down, Fly did not target the LightningRod Pokemon.